### ISSUE TITLE
NTR: regulation of  anther dehiscence (GO:0009901) terms

### ISSUE THREAD

**Issue by lreiser** [geneontology/go-ontology#17128]
Requesting terms for the regulation of a biological process GO:0009901

TERM:negative regulation of anther dehiscence (GO:XXXXXXX)
DEF:Any process that stops, prevents, or reduces the frequency, rate or extent of anther dehiscence

TERM:positive regulation of anther dehiscence (GO:XXXXXXX)
DEF: Any process that activates or increases the frequency, rate or extent of anther dehiscence

REF:PMID:30911018

**Comment by krchristie** [comment]
Hi Lenore,

For logical reasons, I also added the plain regulation term as well as the positive and negative terms you requested. Here's a link to the term stanzas I added:

https://github.com/geneontology/go-ontology/pull/17144/files

In addition to the PMID you supplied, I included you in the definition dbxrefs, assuming that you are GOC:lr. If you've been referred to with a different combination of initials, let me know and I'll fix it.

Thanks,

-Karen
